.github/workflows/terraform.yml

```diff
@@ -104,12 +116,12 @@ jobs:
           output-file: README.md
           output-method: inject
           git-push: true
-          git-commit-message: "docs: update terraform docs [skip ci]"
+          git-commit-message: "docs: update terraform docs"
```

Comment:
I think we want to re-run the CI so the PR is updated with the workflows, otherwise you'd need to check the first commit to see the result.